image_generation/lcm_dreamshaper_v7/cpp/README.md

```diff
@@ -15,7 +15,7 @@ Prepare a python environment and install dependencies:
 ```shell
 conda create -n openvino_lcm_cpp python==3.10
 conda activate openvino_lcm_cpp
-conda install openvino eigen c-compiler cxx-compiler make
+conda install -c conda-forge openvino eigen c-compiler cxx-compiler make
```

Comment:
do we also need to install `openvino-tokenizers` ?

Reply:
Now we're installing openvino-tokenizers from openvino_contrib submodule - not sure if we still need that. 
@Wovchena do we use any other custom operations in other samples?

Reply:
Keep installing from submodule for now. The question of moving all apps to pip openvino-tokenizers is for another pull request